packages/babel-code-frame/src/index.ts

```diff
@@ -1,26 +1,5 @@
-import highlight, { shouldHighlight } from "@babel/highlight";
-
-import _colors, { createColors } from "picocolors";
-import type { Colors, Formatter } from "picocolors/types";
-// See https://github.com/alexeyraspopov/picocolors/issues/62
-const colors =
-  typeof process === "object" &&
-  (process.env.FORCE_COLOR === "0" || process.env.FORCE_COLOR === "false")
-    ? createColors(false)
-    : _colors;
-
-const compose: <T, U, V>(f: (gv: U) => V, g: (v: T) => U) => (v: T) => V =
-  (f, g) => v =>
-    f(g(v));
-
-let pcWithForcedColor: Colors = undefined;
-function getColors(forceColor: boolean) {
-  if (forceColor) {
-    pcWithForcedColor ??= createColors(true);
-    return pcWithForcedColor;
-  }
-  return colors;
-}
+import { getDefs, isColorSupported } from "./defs.ts";
+import { highlight } from "./highlight.ts";
```

Comment:
Can we also export the `highlight` function? So that for `@babel/highlight` users, the can migrate to the `highlight` imported from `@babel/code-frame` when upgrading to Babel 8.